src/main/java/org/folio/rmapi/model/CustomerResources.java

```diff
@@ -0,0 +1,160 @@
+package org.folio.rmapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CustomerResources {
+
+  @JsonProperty("titleId")
+  private String titleId;
+
+  @JsonProperty("packageId")
+  private Integer packageId;
+
+  @JsonProperty("packageName")
+  private String packageName;
+
+  @JsonProperty("isPackageCustom")
+  private Boolean isPackageCustom;
+
+  @JsonProperty("vendorId")
+  private Integer vendorId;
+
+  @JsonProperty("vendorName")
+  private String vendorName;
+
+  @JsonProperty("locationId")
+  private Integer locationId;
+
+  @JsonProperty("isSelected")
+  private Boolean isSelected;
+
+  @JsonProperty("isTokenNeeded")
+  private Boolean isTokenNeeded;
+
+  @JsonProperty("packageType")
+  private String packageType;
+
+  @JsonProperty("visibilityData")
+  private VisibilityInfo visibilityData;
+
+  @JsonProperty("managedCoverageList")
+  private List<CoverageDates> managedCoverageList;
+
+  @JsonProperty("customCoverageList")
+  private List<CoverageDates> customCoverageList;
+
+  @JsonProperty("coverageStatement")
+  private String coverageStatement;
+
+  @JsonProperty("managedEmbargoPeriod")
+  private EmbargoPeriod managedEmbargoPeriod;
+
+  @JsonProperty("customEmbargoPeriod")
+  private EmbargoPeriod customEmbargoPeriod;
+
+  @JsonProperty("url")
+  private String url;
+
+  @JsonProperty("userDefinedField1")
```

Comment:
Similarly these "userDefinedFields" are not in the above sample response in documentation - do we get them in requests to mod-kb(Ruby)? If yes, then probably that doc is incomplete?

Reply:
This comment can be ignored too given that this is coming from RM API.